sos/report/plugins/block.py

```diff
@@ -19,6 +19,9 @@ class Block(Plugin, IndependentPlugin):
     files = ('/sys/block',)
 
     def setup(self):
+        _subdir1 = "udevadm_info_block"
+        _subdir2 = "fdisk_and_parted"
```

Comment:
For code readiness, could be the variable names be more descriptive, like e.g. `udevadm_dir` or `fdisk_dir`?